src/utilities/geometry/Point3d.hpp

```diff
@@ -27,11 +27,11 @@ class UTILITIES_API Point3d
   /// constructor with x, y, z
   Point3d(double x, double y, double z);
 
-  // Copy and move operators are implicitly declared
-  // Point3d(const Point3d& other) = default;
-  // Point3d(Point3d&& other) = default;
-  // Point3d& operator=(const Point3d&) = default;
-  // Point3d& operator=(Point3d&&) = default;
+  // Copy and move operators are implicitly declared, but we want the copy ctor for SWIG so we have to define all of them
+  Point3d(const Point3d& other) = default;
+  Point3d(Point3d&& other) noexcept = default;
+  Point3d& operator=(const Point3d&) = default;
+  Point3d& operator=(Point3d&&) noexcept = default;
```

Comment:
Put back the copy ctor for swig. Have to declare all of them or the move operations are implicitly deleted